packages/manager/modules/vps/src/upscale/upscale.html

```diff
@@ -5,7 +5,7 @@
     dismissable="true"
     on-dismiss="$ctrl.errorMessage = null;"
 >
-    <span data-ng-bind="$ctrl.errorMessage"></span>
+    <span data-ng-bind-html="$ctrl.errorMessage"></span>
```

Comment:
Why do you need `data-ng-bind-html` here?

Reply:
We need to display `<a>` in the error message. So I am using `ng-bind-html`.